Stellar-contract.x

```diff
@@ -255,6 +258,63 @@ struct Int128Parts {
     uint64 hi;
 };
 
+struct ContractInvocation
+{
+    Hash contractID;
+    SCSymbol functionName;
+};
+
+struct AuthorizedInvocation
+{
+    ContractInvocation callStack<>;
+    SCVec topArgs;
```

Comment:
What's the significance of `top`? Is the idea that this contains all args, not just the args for a specific invocation?

Reply:
These are args for the top-of-the-call-stack contract invocation. Maybe it could be named more clearly.

Reply:
I don't think it's the name of this specific field, but rather I'm still piecing together how this works as a whole.